src/Sylius/Bundle/UserBundle/Controller/UserController.php

```diff
@@ -62,22 +63,26 @@ public function changePasswordAction(Request $request)
 
     public function requestPasswordResetTokenAction(Request $request)
     {
-        $generator = $this->container->get('sylius.user.token_provider');
+        $generator = $this->container->get('sylius.user.password_reset.token_provider');
```

Comment:
I know that it sounds weird but in sylius we got convention something like ``sylius.user.type_of_service.name``.